themes/meghna-hugo/assets/css/custom.css

```diff
@@ -9,4 +9,8 @@ blockquote {
 
 blockquote p {
   display: inline;
+}
+
+.post-single-content img {
+  width: 100%;
```

Comment:
When I was reading the blog post the external figures were much wider than the post content and overflowing horizontally. This CSS directive ensures all of the inline post images are the same width as the text.